dask_ml/model_selection/_incremental.py

```diff
@@ -287,6 +304,8 @@ def fit(
     fit_params=None,
     scorer=None,
     random_state=None,
+    verbose=1,
```

Comment:
Can you make the default here match the others (False)?

Reply:
I've renamed `verbose` to `log_freq` to make it more clear.